src/platform/plugins/shared/saved_objects_management/public/management_section/objects_table/components/table.tsx

```diff
@@ -369,26 +373,77 @@ export class Table extends PureComponent<TableProps, TableState> {
       );
     }
 
-    const button = (
-      <EuiButton
-        iconType="arrowDown"
-        iconSide="right"
-        onClick={this.toggleExportPopoverVisibility}
-        isDisabled={selectedSavedObjects.length === 0}
-      >
-        <FormattedMessage
-          id="savedObjectsManagement.objectsTable.table.exportPopoverButtonLabel"
-          defaultMessage="Export"
-        />
-      </EuiButton>
-    );
-
     const activeActionContents = this.state.activeAction?.render() ?? null;
     const exceededResultCount = totalItemCount > MAX_PAGINATED_ITEM;
 
+    const hasMlObjects = selectedSavedObjects.some(({ type }) => ML_SAVED_OBJECT_TYPES.has(type));
+
     const anySelected = selectedSavedObjects.length > 0;
     const allHidden =
       anySelected && selectedSavedObjects.every(({ meta: { hiddenType } }) => hiddenType);
+
+    const deleteTooltip = () => {
+      if (hasMlObjects) {
+        return (
+          <FormattedMessage
+            id="savedObjectsManagement.objectsTable.table.hasMlObjects.deleteDisabledTooltip"
+            defaultMessage="Navigate to the Machine Learning management pages to delete machine learning jobs."
```

Comment:
This needs to change depending on whether trained models and / or jobs are selected.